nucleus/admin/config-api/src/main/java/com/sun/enterprise/config/modularity/ConfigModularityUtils.java

```diff
@@ -935,9 +934,6 @@ public String getRuntimeTypePrefix(StartupContext startupContext) {
             ConfigInjector injector = serviceLocator.getService(ConfigInjector.class, desc.getName());
             if (injector != null) {
                 String clzName = injector.getClass().getName().substring(0, injector.getClass().getName().length() - 8);
-                if (clzName == null) {
```

Comment:
This condition is needed. If we remove this, the below statement will throw exception